src/screens/HyperSwitch/RoutingRevamp/AdvancedRoutingUtils.res

```diff
@@ -321,3 +330,69 @@ let getRoutingTypesFromJson = (values: Js.Json.t) => {
 let validateStatements = statementsArray => {
   statementsArray->Js.Array2.every(isStatementMandatoryFieldsPresent)
 }
+
+let generateStatements = statements => {
+  statements->Array.reduce(
+    [
+      {
+        condition: [],
+      },
+    ],
+    (acc, statement) => {
+      let statementDict = statement->LogicUtils.getDictFromJsonObject
```

Comment:
open `LogicUtils` at the start of the function and remove in other places.

Reply:
Opened LogicUtils at the start of the page